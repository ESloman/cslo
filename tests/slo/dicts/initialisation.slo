var map = {
    "key": "value",
    "key1": "value1",
    "key2": "value2",
    "key": 1
};

println(map);
