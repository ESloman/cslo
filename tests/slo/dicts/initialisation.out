dict[3]: {key2: value2, key: 1, key1: value1}
